src/app/core/components/language-picker/language-picker.component.ts

```diff
@@ -28,16 +27,16 @@ export class LanguagePickerComponent implements OnInit, OnChanges {
     this.initCurrentLang();
   }
 
-  languageChanged(lang: { value: { tag: string } }): void {
-    this.changeLang.emit(lang.value.tag);
+  languageChanged(lang: { value: string }): void {
+    this.changeLang.emit(lang.value);
 
     if (this.redirectOnChange) {
-      this.localeService.navigateTo(lang.value.tag);
+      this.localeService.navigateTo(lang.value);
     }
   }
 
   private initCurrentLang(): void {
-    this.current = this.defaultLang ? this.languages.find(({ tag }) => tag === this.defaultLang) : this.localeService.currentLang;
+    this.current = this.defaultLang || this.localeService.currentLang?.tag;
```

Comment:
In TS, should be `this.defaultLang ?? this.localeService.currentLang?.tag;`

Reply:
Fixed.